app/src/modules/vega-chart.js

```diff
@@ -93,4 +93,17 @@ function buildSparqlSpec (baseSpec, sparqlResults) {
   return spec
 }
 
-export { getDefaultChart, loadChart, buildSparqlSpec }
+const chartUriPrefix = 'http://nanomine.org/viz/'
+
+function toChartId(chartUri) {
+  if (!chartUri.startsWith(chartUriPrefix)) {
+    throw new Error(`Unexpected chart uri "${chartUri}". Was expecting prefix "${chartUriPrefix}"`)
+  }
+  return chartUri.substring(chartUriPrefix.length)
+}
+
+function toChartUri(chartId) {
+  return chartUriPrefix + chartId
+}
+
+export { getDefaultChart, loadChart, buildSparqlSpec, toChartId, toChartUri, chartUriPrefix }
```

Comment:
This is subjective, but I would think these should be component-based methods since they are small lines of codes and are not reusable elsewhere. With that, we can then proceed to remove its import in test (Gallery.spec.js), and should still work.

Reply:
Thanks for the clarification